packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useBreadcrumbsStyleProps } from './useBreadcrumbsStyleProps';
+import { SpiritBreadcrumbsProps } from '../../types';
+import { useStyleProps } from '../../hooks/styleProps';
+
+const defaultProps = {
+  items: [],
+};
+
+export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcrumbsProps<T>): JSX.Element => {
+  const { elementType: ElementTag = 'nav', ...restProps } = props;
+  const { classProps, props: modifiedProps } = useBreadcrumbsStyleProps(restProps);
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  return (
+    <ElementTag
+      {...otherProps}
+      {...styleProps}
+      className={classNames(classProps, styleProps.className)}
+      aria-label="Breadcrumb"
+    >
+      <ol>
+        {restProps.items.map((item, index) => (
+          <React.Fragment key={`BreadcrumbsItem_${item.url}`}>
+            {index === 0 && (
+              <li className="d-tablet-none">
+                <a href={item.url}>Back</a>
+              </li>
+            )}
+            <li className="d-none d-tablet-flex">
+              <a href={item.url} aria-current={index === restProps.items.length - 1 ? 'page' : undefined}>
```

Comment:
I am thinking about making this condition more readable and defining an arrow function for this. But I am quite not sure if it is necessary. What is your opinion?

```suggestion
              <a href={item.url} aria-current={isCurrentPage(index, itemsCount) ? 'page' : undefined}>
```

Reply:
Since you think this is not a very nice markdown for you, I suggest to make a function `isLast` because that's what it's supposed to say, not that the page is a current page. There's no url path and breadcrumb url item comparison. Do you agree?